tdp/cli/commands/browse.py

```diff
@@ -173,11 +177,6 @@ def custom_format(key, value):
                 return value[0] + ",...," + value[-1]
             else:
                 return ",".join(value)
-        elif key == "operations":
-            if len(value) > 2:
-                return value[0].operation + ",...," + value[-1].operation
-            else:
-                return ",".join(operation_log.operation for operation_log in value)
```

Comment:
If we keep the code as it was in `process_single_deployment_query`, we should keep this.